src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/telemetry/telemetry.component.html

```diff
@@ -207,7 +207,7 @@
               <button type="button"
                       class="btn btn-light"
                       (click)="next()">
-                <ng-container i18n>{{ actionLabels.NEXT }}</ng-container>
+                <ng-container i18n>Next</ng-container>
```

Comment:
You should better keep the original code.
```suggestion
                <ng-container>{{ actionLabels.NEXT }}</ng-container>
```
But i think you can remove the `i18n` argument because the label is already translated [here](https://github.com/ceph/ceph/blob/master/src/pybind/mgr/dashboard/frontend/src/app/shared/constants/app.constants.ts#L143).

Reply:
Good note! Change effected.